js_modules/dagit/packages/core/src/instance/BackfillTable.tsx

```diff
@@ -179,47 +176,13 @@ const BackfillRow = ({
     },
   ]);
 
-  const repoAddress = backfill.partitionSet
```

Comment:
Ahh man I edited this today in https://github.com/dagster-io/dagster/pull/9737 too, let's merge this one first and I think I can apply my changes easily.